src/safeds/data/tabular/containers/_table.py

```diff
@@ -2380,3 +2382,41 @@ def __dataframe__(self, nan_as_null: bool = False, allow_copy: bool = True):  #
         data_copy = self._data.reset_index(drop=True)
         data_copy.columns = self.column_names
         return data_copy.__dataframe__(nan_as_null, allow_copy)
+
+    def into_dataloader(self, batch_size: int) -> DataLoader:
```

Comment:
this should be a private function, as the user will not use this function and work directly on Torch objects.

```_into_dataloader``` or mby add something like a neural network utils